FirebaseAuth/Sources/Auth/FIRAuth_Internal.h

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
+#import <FirebaseAuth/FIRAuth.h>
```

Comment:
Isn't this a circular dependency for building the library itself?  Does this work in a clean build?

Reply:
> Isn't this a circular dependency for building the library itself?

My understanding is that the answer is no because the <...> header will first try to resolve from a framework in the systems directory. In this case, the framework does not depend on a framework called "FirebaseAuth" so the header is then attempted to be resolved as a textual import (e.g. "...") resolved from the public headers path.

> Does this work in a clean build?

Yes.